pkgs/data/themes/catppuccin-sddm/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  qtgraphicaleffects,
+  qtsvg,
+  qtquickcontrols2,
+  flavor ? "mocha",
+  font ? "Noto Sans",
+  fontSize ? "9",
+  background ? null,
+  loginBackground ? false,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "catppuccin-sddm";
+  version = "main-2024-03-17";
```

Comment:
Version 1.0.0 just got released https://github.com/catppuccin/sddm/releases

Reply:
Updated to 1.0.0, thank you for pointing this out.